### PR TITLE
Website rewrite from Jekyll to vanilla HTML/CSS

### PR DESCRIPTION
**Pull request by cuppajoe123**
This pull request contains an entire rewrite of our existing website. All of the relevant content is the same however. The reasoning behind this rewrite is that the static site generators that we were using proved tiresome and difficult to set up. Jekyll, the one we used previously, required Ruby and many packages from its repositories, which can and will result in version mix-ups and dependency conflicts, especially if one hasn't update their Ruby Gem repositories for long periods of time. Jekyll also provided many features that we just didn't use, thus adding necessary abstraction and complexity. In contrast, HTML and CSS do not require any such toolchain setup and can be edited and tested on any computer with a browser. They are also ubiquitous on the web and useful skills to boot.

The CSS file I created was adapted from my own website, but with several changes in spacing and colors. I chose darker, softer colors that I think go nicely together. Of course, these can be changed later. Let me know your thoughts :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      </ul>
+      <li>Join the CSC <a href="https://discord.gg/kUd3m8HMG8">Discord Server</a></li>
+      <li>(coming soon) Join our IRC channel at #bfhs-open on libera.chat</li>
+    </ul>
+
+    <h1>How to participate</h1>
+    <ul>
+      <li><a href="https://github.com/BFHS-Open">Contribute to our various projects on Github</a></li>
+      <li><a href="https://github.com/BFHS-Open/knowledge-base">Create your own knowledge base and share it with us</a></li>
+      <li><a href="https://bfhs-open.github.io/ctf/">Enter into competitions with us</a></li>
```
Comment (Aeshus) [inline index.html]: Does the /ctf/ directory currently point anywhere?
Seems like the `ctf.md` page was deleted and there isn't any proposed html pages to replace it?

File: README.md [hunk 2]
```diff
@@ -2,11 +2,7 @@
...
 ## Resources
-- [Jekyll Documentation](https://jekyllrb.com/docs/) - Please become familiar with how Jekyll works and how to use it <br />
-- [Basic Markdown Syntax](https://www.markdownguide.org/basic-syntax/)
-
-## TODO
-- [Decide on theme + layout](https://jekyllrb.com/docs/themes/)
-- Home page
-- About page
+- [The Odin Project Foundations](https://www.theodinproject.com/paths/foundations/courses/foundations): Basics of HTML, CSS, and beyond
+- [MDN Docs](https://developer.mozilla.org/en-US/): In-depth web development documentation
```
Comment (Aeshus) [inline README.md:10]:
```suggestion
- [HTML 5 Specifications](https://www.w3.org/TR/html5/): HTML Specification

```

Comment (cuppajoe123) [reply README.md:10]: Not HTML5?

Comment (Aeshus) [reply README.md:10]: didn't expect you to actually read it ;)

File: index.html (hunk 1 above)
Comment (cuppajoe123) [reply index.html]: I was looking at the Competitions page, and it read as though it was an upcoming event. Do you think we should rewrite it's content, and include it in this pull request, or remove that link and save it for a future pull request?

File: README.md [hunk 3]
```diff
@@ -2,11 +2,7 @@
 Website for the BFHS-Open organization, maintained by the BFHS Computer Science Club
```
Comment (Aeshus) [inline README.md:3]:
```suggestion

This is meant as a way to involve BFHS Students in contributing to open source projects, while also giving more visibility to the open source activities done by our members.

```

More flavor for our mission statement.

File: README.md (hunk 2 above)
Comment (cuppajoe123) [reply README.md:10]: Ahh yes, just blindly merging the pull request without reading the suggestions.

File: index.html (hunk 1 above)
Comment (Aeshus) [reply index.html]: Yeah I would probably remove it for the time being and reintroduce it at a later date.

Also honestly, we aren't really _looking_ for new members, just merely showing off our prior competitions.

```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1,12 +1,11 @@
...
+This is meant as a way to involve BFHS Students in contributing to open source projects, while also giving more visibility to the open source activities done by our members.
 
-## TODO
-- [Decide on theme + layout](https://jekyllrb.com/docs/themes/)
-- Home page
-- About page
+## Resources
+- [The Odin Project Foundations](https://www.theodinproject.com/paths/foundations/courses/foundations): Basics of HTML, CSS, and beyond
+- [MDN Docs](https://developer.mozilla.org/en-US/): In-depth web development documentation
+- [HTML 5 Specifications](https://www.w3.org/TR/html5/): HTML Specification
```
Comment (Aeshus) [inline README.md]: Sounds better to put it like this. Other than that that, all good!

```suggestion
- [HTML Specs](https://www.w3.org/TR/html5/): W3's HTML 5 Specification
```

**Review by Aeshus — APPROVED** [review]
## Looks good to me!
Hopefully it doesn't break everything!

**Pull request merged**